app/src/lib/git/branch.ts

```diff
@@ -156,3 +156,18 @@ export async function getBranchesPointedAt(
   // split (and remove trailing element cause its always an empty string)
   return stdout.split('\n').slice(0, -1)
 }
+
+export async function getMergedBranches(
+  repository: Repository,
+  branchName: string
+): Promise<ReadonlyArray<string>> {
+  const args = ['branch', '--merged', branchName]
+
+  const { stdout } = await git(args, repository.path, 'mergedBranches')
```

Comment:
I think we should use a similar approach here as we do in `getBranches`, i.e. `git branch --format="%(refname:short)"`. That way we're certain that we're getting the same values from `getBranches` and `getMergedBranches` and it also saves us from having to do string trimming.